<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickMosaicClass</class>
 <widget class="QDialog" name="QuickMosaicClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuickMosaic</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>基本设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>影像路径：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>格式：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="image_path">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>34</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="imageFormat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>*.jpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*.tif</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>33</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="quickmosaic.qrc"/>
 </resources>
 <connections/>
</ui>
